lib/commands/init.js

```diff
@@ -44,19 +44,23 @@ async function run(options) {
         return
     }
     log(chalk.bold('Building packages'))
-    try {
-        await runner(
-            'forge-ui-components',
-            `${NPM} run build`,
-            { cwd: path.join(options.packageDir, 'forge-ui-components') }
-        )
-        log(`${chalk.greenBright('+')} flowforge/forge-ui-components`)
-    } catch(err) {
-        log(`${chalk.greenBright('-')} flowforge/forge-ui-components`)
-        error('Failed to build forge-ui-components')
-        error(err.stderr)
-        error('Aborting')
-        return
+    const buildPackages = ['forge-ui-components','flowforge']
```

Comment:
I wonder if build should be split out of `init` so one can invoke it after pulling new data? `init` itself invokes a subtask build then?

Mostly trying to pull `build` up to this repo as I think just building doesn't fit the `init` verb as command.

Reply:
> I wonder if build should be split out

Yes - I considered that, but decided this was a good enough change for now.

When in 'development' mode, developers will typically be running `npm run serve` in the `flowforge` repo which automatically rebuilds the code - but you need to have done a one-off `npm run build` first. This is why it felt appropriate to be in the `init` task as a one-off thing.

It really depends on how we expect devs to work. I'm wary of hoisting lots of tasks out of individual repos up into this dev repo, but I'm sure there will be some more common things that make sense to break out.